packages/core/src/base-axis-chart.ts

```diff
@@ -244,6 +245,15 @@ export class BaseAxisChart extends BaseChart {
 				.text(this.options.scales.x.title);
 		}
 
+		// Add chart title
+		if (this.options.scales.headers.title) {
+			this.innerWrap.append("text")
+				.attr("class", "title")
+				.attr("text-anchor", "middle")
+				.attr("transform", `translate(${xAxisRef.node().getBBox().width / 2}, 0)`)
+				.text(this.options.scales.headers.title);
```

Comment:
The chart would also need to be pushed down when user adds a title. Currently the chart stays in the same position. Design specs need to be reviewed for the aesthetic implementation.